apps/omg_watcher_rpc/lib/web/router.ex

```diff
@@ -27,22 +27,32 @@ defmodule OMG.WatcherRPC.Web.Router do
     post("/account.get_utxos", Controller.Account, :get_utxos)
     post("/account.get_exitable_utxos", Controller.Account, :get_exitable_utxos)
 
+    # security
     post("/in_flight_exit.get_data", Controller.InFlightExit, :get_in_flight_exit)
+    # security
     post("/in_flight_exit.get_competitor", Controller.InFlightExit, :get_competitor)
+    # security
     post("/in_flight_exit.prove_canonical", Controller.InFlightExit, :prove_canonical)
+    # security
     post("/in_flight_exit.get_input_challenge_data", Controller.InFlightExit, :get_input_challenge_data)
+    # security
     post("/in_flight_exit.get_output_challenge_data", Controller.InFlightExit, :get_output_challenge_data)
 
     post("/transaction.all", Controller.Transaction, :get_transactions)
     post("/transaction.get", Controller.Transaction, :get_transaction)
+    # security
     post("/transaction.submit", Controller.Transaction, :submit)
     post("/transaction.submit_typed", Controller.Transaction, :submit_typed)
     post("/transaction.create", Controller.Transaction, :create)
 
+    # security
     post("/utxo.get_exit_data", Controller.Utxo, :get_utxo_exit)
+    # security
     post("/utxo.get_challenge_data", Controller.Challenge, :get_utxo_challenge)
 
+    # security
     post("/status.get", Controller.Status, :get_status)
+    # security
```

Comment:
Would it be better to just create a section just for the `security` routes instead of commenting on each portion?